### ISSUE TITLE
508-defect-2 [COGNITION, SEMANTIC MARKUP] The link text MUST make sense in context, and should make sense when taken out of context

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#16049]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Point of Contact

**VFS Point of Contact:** Josh Kim

## User Story or Problem Statement

As a user with cognitive considerations and/or as a user with a screen reader, I expect the purpose of the link to be clear from the link text. 

## Details

Currently, there is a link within an additional details grey box for when a user selects "a claim other than disability compensation." If a screen reader user were to tab, they would hear "link, benefit office" which on its own does not clearly explain its purpose. This would force a screen reader user to have to navigate around the link itself in order to understand its purpose.

Providing more details within the link itself that clearly explains its purpose makes navigating the page significantly easier for these users.

## Acceptance Criteria

- [ ] Link text is updated to "Send the completed form to the benefit office"

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Environment

* Operating System: All 
* Browser: All
* Screenreading device: All
* Server destination: Staging

## Steps to Recreate

1. Enter `<https://staging.va.gov/decision-reviews/higher-level-review/request-higher-level-review-form-20-0996/introduction>` in browser
2. Select "a claim other than disability compensation"
3. Verify the grey alert box only has the link wrap around "benefit office"

### Current
```Diff
<p class="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-top--2">
  You’ll need to fill out and submit VA form 20-0996 by mail or in person. 
  <!-- Link only wraps benefit office -->
!  Send the completed form to the <a href="/decision-reviews/higher-level-review/#8622">benefit office</a> that matches the benefit type you select on the form.
!  <!-- Missing closing paragraph tag -->
<p class="vads-u-margin-bottom--0">
  <a href="https://www.vba.va.gov/pubs/forms/VBA-20-0996-ARE.pdf" download="VBA-20-0996-ARE.pdf" type="application/pdf">

  <i aria-hidden="true" class="fas fa-download vads-u-padding-right--1" role="img"></i>

  Download VA Form 20-0996 
  <dfn><abbr title="Portable Document Format">PDF</abbr> 
    (1.5<abbr title="Megabytes">MB</abbr>)</dfn>
  </a>

! </p>  <!-- Here is the closing tag for the first p -->

</p>

```

### Recommendation
```Diff
<p class="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-top--2">
  You’ll need to fill out and submit VA form 20-0996 by mail or in person. 

+  <a href="/decision-reviews/higher-level-review/#8622">Send the completed form to the benefit office</a> that matches the benefit type you select on the form.
+  </p>

<p class="vads-u-margin-bottom--0">
<a href="https://www.vba.va.gov/pubs/forms/VBA-20-0996-ARE.pdf" download="VBA-20-0996-ARE.pdf" type="application/pdf">

  <i aria-hidden="true" class="fas fa-download vads-u-padding-right--1" role="img"></i>

  Download VA Form 20-0996 
  <dfn><abbr title="Portable Document Format">PDF</abbr> 
    (1.5<abbr title="Megabytes">MB</abbr>)</dfn>
  </a>

</p>

```

## WCAG or Vendor Guidance (optional)

* [WCAG 2.0 Level A - 2.4.4 Link Purpose (In Context)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

* [WCAG 2.0 Level A - 2.4.4 Link Purpose (In Context)](https://www.wuhcag.com/link-purpose-in-context/), human readable explanation of success criteria

### Before
<img width="633" alt="Screen Shot 2020-11-11 at 11 14 17 AM" src="https://user-images.githubusercontent.com/14154792/98837283-c77f0880-2410-11eb-8d3e-3571c14000b8.png">

### After
<img width="635" alt="Screen Shot 2020-11-11 at 11 14 53 AM" src="https://user-images.githubusercontent.com/14154792/98837282-c6e67200-2410-11eb-9215-20841471e744.png">

**Comment by joshkimux** [comment]
<img width="876" alt="Screen Shot 2020-12-28 at 11 39 37 AM" src="https://user-images.githubusercontent.com/14154792/103229591-5ed8e600-4901-11eb-92dd-f839bfc57e0a.png">
This has been fixed 👍
